src/FFTW.jl

```diff
@@ -91,4 +91,9 @@ end
 include("fft.jl")
 include("dct.jl")
 
+if Base.VERSION >= v"1.4.2"
```

Comment:
Can you add a comment about why it's this particular version?

Reply:
Done